requirements-dev.txt

```diff
@@ -20,6 +20,11 @@ gitpython
 gitdb
 sphinx
 sphinx-panels
+numpydoc < 1.2
```

Comment:
@pandas-dev/pandas-core do we have an issue about this? can we remove (independently than this PR)

Reply:
#39688